playbulb-candle/playbulbCandle.js

```diff
@@ -114,6 +116,20 @@
         this._isEffectSet = true;
       });
     }
+    startBlowNotifications(listener) {
+      let characteristic = this._characteristics.get(CANDLE_BLOW_NOTIFICATIONS_UUID);
+      return characteristic.startNotifications()
+      .then(_ => {
+        characteristic.addEventListener('characteristicvaluechanged', listener);
+      });
+    }
+    stopBlowNotifications(listener) {
+      let characteristic = this._characteristics.get(CANDLE_BLOW_NOTIFICATIONS_UUID);
+      return characteristic.stopNotifications()
+      .then(_ => {
+        characteristic.removeEventListener('characteristicvaluechanged', listener);
```

Comment:
I had to read your last comment twice ;)

As a developer, if calling `playbulbCandle.stopBlowNotifications(handleBlowNotifications)` rejects, then I expect the listener to not be removed since the operation failed. Does that make sense?